modules/src/clients/ics07_tendermint/client_state.rs

```diff
@@ -99,6 +100,7 @@ impl ClientState {
             max_clock_drift,
             frozen_height,
             latest_height,
+            proof_specs,
```

Comment:
Should we allow empty proof-specs?

Reply:
I am not sure I understand what this would entail? Can you elaborate on what happens if we use empty proof-specs?

Reply:
Thinking about this again - an empty `ProofSpecs` implies that the chain doesn't have a `ProvableStore` and therefore cannot provide proofs for the data it stores. This is a requirement in ICS024, so I don't think we should allow empty proof-specs.

Reply:
Makes sense, agreed!

Reply:
https://github.com/informalsystems/ibc-rs/pull/1574/commits/a229708fdde2510f87913c4f7570bd4634439bb6